src/components/Sidebar/Sidebar.story.tsx

```diff
@@ -21,37 +27,60 @@ const Header = styled(H)`
   margin-top: 0;
 `;
 
-const Template: Story<Props> = ({ attach = 'left', ...args }) => {
+const Template: Story<Props & { labelAsTooltip?: boolean }> = ({
```

Comment:
If you add `labelAsTooltip` to `ItemPropsIntrinsic` you can change this to `Story<Props & Pick<ItemPropsIntrinsic, 'labelAsTooltip'>>` so you can just pass in `args.labelAsTooltip` to the `Sidebar.Item`s below...